doc/Dependency.rst

```diff
@@ -24,12 +24,14 @@ Dependency
 pySVModel Package
 *****************
 
-+-----------------------------------------------------+-------------+---------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------+
-| **Package**                                         | **Version** | **License**                                                         | **Dependencies**                                                                                                                |
-+=====================================================+=============+=====================================================================+=================================================================================================================================+
-| `pydecor <https://github.com/mplanchard/pydecor>`__ | ≥2.0.1      | `MIT <https://github.com/mplanchard/pydecor/blob/master/LICENSE>`__ | * `dill <https://github.com/uqfoundation/dill>`__ (`BSD 3-clause <https://github.com/uqfoundation/dill/blob/master/LICENSE>`__) |
-|                                                     |             |                                                                     | * `six <https://github.com/benjaminp/six>`__ (`MIT <https://github.com/benjaminp/six/blob/master/LICENSE>`__)                   |
-+-----------------------------------------------------+-------------+---------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------+
+*None*
```

Comment:
Should be replaced by `pyTooling` and use `@export` from there.

Reply:
The `@export` from pyTooling is used already, according to https://github.com/edaa-org/pySVModel/pull/8/commits/a647a8aa8e007f0bf4275d82e760915119b64178#diff-0dafa286111e7584f493ad2c0217e04bc7f137256070dc0ef549a43c8c0d9ced.

Reply:
But it's not documented yet. This diff shows not the matching entry.

Reply:
@umarcor see this PR: https://github.com/edaa-org/pySVModel/pull/16 and review. If OK for you merge it and this one should be also done and can be merged (by you as all other is then fixed or delayed).